build/tutorials/platform/integrate-exchange-with-avalanche.md

```diff
@@ -2,7 +2,9 @@
 
 ## Overview
 
-The objective of this document is to provide a brief overview of how to integrate with the EVM-Compatible Avalanche C-Chain. For teams that already support ETH, supporting the C-Chain is as straightforward as spinning up an Avalanche node \(which has the [same API](https://eth.wiki/json-rpc/API) as [go-ethereum](https://geth.ethereum.org/docs/rpc/server)\) and populating Avalanche’s ChainID \(43114\) when constructing transactions.
+The objective of this document is to provide a brief overview of how to integrate with the EVM-Compatible Avalanche C-Chain. For teams that already support ETH, supporting the C-Chain is as straightforward as spinning up an Avalanche node \(which has the [same API](https://eth.wiki/json-rpc/API) as [go-ethereum](https://geth.ethereum.org/docs/rpc/server)\) and populating Avalanche’s ChainID \(43114\) when constructing transactions. There is also a [Rosetta](https://www.rosetta-api.org/) implementation for Avalanche, which can be even easier to integrate.
+
+## Direct integration with C-Chain
```

Comment:
nit: `Integration using EVM Endpoints` (or something like that to stay consistent with other titles...I think `Direct Integration` is confusing)